src/ctia/stores/es/crud.clj

```diff
@@ -202,24 +213,125 @@ It returns the documents with full hits meta data including the real index in wh
   [docs ident get-in-config]
   (filter #(allow-read? % ident get-in-config) docs))
 
+(s/defschema BulkResult
+  (st/optional-keys
+   {:deleted [s/Str]
+    :updated [s/Str]
+    :errors (st/optional-keys
+             {:forbidden [s/Str]
+              :not-found [s/Str]
+              :internal-error [s/Str]})}))
+
+(defn- format-bulk-res
+  [bulk-res]
```

Comment:
This assume a certain contstrained format, for documentation sake it would probably be nice to add a few schemas to this definition.

Even nicer, I think a docstring with an example of expected value would be nice.
For example the `(group-by :result)` is not really natural to follow for me.